.buildconfig.yml

```diff
@@ -232,6 +232,10 @@ projects:
     path: components/service/glean
     description: 'A client-side telemetry SDK for collecting metrics and sending them to the Mozilla telemetry service'
     publish: true
+  service-glean-gradle-plugin:
```

Comment:
namespace nit: `service` in AC relates to a "web service" or anything talking to servers. Technically I think this would fit better into our `tooling` category that hosts things that are not really components themselves (e.g. `tooling-lint` for custom lint checks). However we could consider creating a `plugins` category because as said before there is some interest in providing other plugins!